integrations/crowdnode/src/main/java/org/dash/wallet/integrations/crowdnode/api/CrowdNodeApi.kt

```diff
@@ -193,34 +200,76 @@ class CrowdNodeBlockchainApi @Inject constructor(
         }
     }
 
+    override suspend fun withdraw(amount: Coin): Boolean {
+        val accountAddress = this.accountAddress
+        requireNotNull(accountAddress) { "Account address is null, make sure to sign up" }
+
+        val balance = this.balance.value.data ?: Coin.ZERO
+        require(amount <= balance) { "Amount is larger than CrowdNode balance" }
+
+        return try {
+            apiError.value = null
+
+            val maxPermil = ApiCode.WithdrawAll.code
+            val requestPermil = min(amount.value * maxPermil / balance.value, maxPermil)
```

Comment:
Is "mil" 1000th or 0.1%?

Reply:
One per mil is 1/1000 or 0.1%: https://en.wikipedia.org/wiki/Per_mille